public/index.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express");
+const path = require("path");
+const mongoose = require("mongoose");
+const multer = require("multer");
+require("dotenv").config();
+const { upload } = require("./diskStorage");
+
+const { DB_HOST: urlDb } = process.env;
+const uploadApp = express();
+
+uploadApp.use(express.json());
+
+uploadApp.use(express.static(path.join(__dirname, "../public")));
+
+uploadApp.post("/avatars", upload.single("picture"), (req, res) => {
```

Comment:
Niezabezpieczona ścieżka dodaje zdjęcia. Potrzebujesz jej do czegoś w przyszłości? Bo jeśli dla testów ją napisałaś to usuń. Ta jedna która przypisuje avatar do uzytkownika wystarczy. Dzięki temu masz kontrole co jest dodawane.